src/definitions/elements/list.less

```diff
@@ -107,6 +107,7 @@ ol.ui.list ol li,
 .ui.list .list > .item > i.icon,
 .ui.list > .item > i.icon {
   display: table-cell;
+  min-width: 1.55em;
```

Comment:
Could you please move the value into a variable?

Reply:
I also thought about this, I don't think it should have a variable I think it should work it out from the icon font size so therefore it will change if you change your icon size.

Reply:
OK, agreed